Doc/library/datetime.rst

```diff
@@ -653,46 +653,46 @@ Supported operations:
 | Operation                     | Result                                       |
 +===============================+==============================================+
 | ``date2 = date1 + timedelta`` | ``date2`` will be ``timedelta.days`` days    |
-|                               | after ``date1``. (1)                         |
+|                               | after ``date1``. [#date1]_                   |
```

Comment:
A possible negative is that footnotes  are numbered sequentially throughout the whole file instead of restarting with 1 (because of the explicit number labels starting with 1).  This, for instance, becomes file footnote 7 rather that table note 1.
>date2 = date1 + timedelta | date2 will be timedelta.days days after date1. [7]

in the preview.    On the other hand, The links to notes and backlinks from note to table line are nice, especially when multiple, as in
>[1] (1,2,3,4)  This is exact but may overflow.

where 1,2,3,4 link to the 4 items with this property.  This is note such a big deal on my 4K monitor, both I imagine would be very helpful with phones, and I know there are coredevs and I presume other users that use phones.  Overall, I think the change is positive.